### PR TITLE
ENG-29 get_available_jobs_count should take all pods into account

### PR DESCRIPTION
**Pull request by zubenkoivan**

### CODE REVIEW

File: platform_monitoring/kube_client.py [hunk 1]
```diff
@@ -619,32 +767,44 @@ async def create_pod_container_logs_stream(
...
             params["labelSelector"] = label_selector
         if field_selector:
             params["fieldSelector"] = field_selector
+        payload = await self._request(method="get", url=url, params=params)
+        self._assert_resource_kind("PodList", payload)
+        pod_list = ListResult.from_primitive(payload, resource_cls=Pod)
+        return pod_list.items
         payload = await self._request(method="get", url=self._pods_url, params=params)
         self._assert_resource_kind("PodList", payload)
         return [Pod(p) for p in payload["items"]]
```
Comment (zubenkoivan) [inline platform_monitoring/kube_client.py:788]: locally these lines are deleted

Comment (YevheniiSemendiak) [reply platform_monitoring/kube_client.py:790]: this is github's glitch?

**Review by YevheniiSemendiak — APPROVED** [review]
Great job! :+1:

**Comment by zubenkoivan** [comment]
@YevheniiSemendiak yes, there are some github glitches or may be even git in few places

**Pull request merged**